Makefile

```diff
@@ -378,8 +378,13 @@ ifeq ($(HV),kubevirt)
 else
         #kube container will not be in non-kubevirt builds
         PKGS_$(ZARCH)=$(shell find pkg -maxdepth 1 -type d | grep -Ev "eve|alpine|sources|kube|external-boot-image|verification$$")
+# nvidia platform requires more space
```

Comment:
It would be nice to indent it for readability.

Reply:
I would suggest keeping the same style as in the rest of the file and maybe addressing the entire file later, but definitely not in this PR...

Reply:
makes sense. it's done.